packages/react-devtools-shared/src/config/DevToolsFeatureFlags.core-fb.js

```diff
@@ -29,7 +29,5 @@ export const enableProfilerComponentTree = true;
 import typeof * as FeatureFlagsType from './DevToolsFeatureFlags.default';
 import typeof * as ExportsType from './DevToolsFeatureFlags.core-fb';
 
-// eslint-disable-next-line no-unused-vars
-type Check<_X, Y: _X, X: Y = _X> = null;
-// eslint-disable-next-line no-unused-expressions
-(null: Check<ExportsType, FeatureFlagsType>);
+// Flow magic to verify the exports of this file match the original version.
```

Comment:
Have you verified this still does what it's supposed to?

Reply:
I changed one export type and it showed an error.

The idea is that A can be casted into B and B can be casted into A, which I think implies that A = B.